packages/quantic/force-app/examples/main/lwc/exampleQuanticDocumentSuggestion/exampleQuanticDocumentSuggestion.js

```diff
@@ -17,6 +19,12 @@ export default class ExampleQuanticDocumentSuggestion extends LightningElement {
       description: 'The id of the case assit engine associated.',
       defaultValue: 'case-assist-engine',
     },
+    {
+      attribute: 'searchHub',
```

Comment:
I think we should not include the `searchHub` option in this form. The reason is that the option is being set on the interface component, and not on the tested component (`QuanticDocumentSuggestion` in this case).

Why not simply affect the `searchHub` attribute a static value?

Reply:
You are right! I just updated this thank you👌